receiver/databricksreceiver/go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/databricksreceiver
+
+go 1.19
+
+require (
+	github.com/stretchr/testify v1.8.2
+	go.opentelemetry.io/collector v0.76.1
+	go.opentelemetry.io/collector/component v0.76.1
+	go.opentelemetry.io/collector/confmap v0.76.1
+	go.opentelemetry.io/collector/pdata v1.0.0-rcv0011
+	go.opentelemetry.io/collector/receiver v0.76.1
```

Comment:
Please rebase and update versions here.

Reply:
@atoulme do you mind if I close this PR and hand this work off to another team member?